src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/VariableDeclarationUsageDistanceCheck.java

```diff
@@ -692,10 +692,10 @@ else if (variableUsageExpressions.isEmpty()) {
                 if (isChild(currentStatementAst, variableAst)) {
                     variableUsageExpressions.add(currentStatementAst);
                 }
-                // If expression doesn't contain variable and this variable
+                // If expression is not zero-distance token and this variable
                 // hasn't been met yet, then distance + 1.
```

Comment:
We can remove this comment, new method name and clear logic make it obvious what is going on here.

Reply:
Done